extensions/amp-image-lightbox/0.1/test/integration/test-amp-image-lightbox.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {poll} from '../../../../../testing/iframe';
+
+describe.configure().run('amp-image-lightbox', function() {
+  const extensions = ['amp-image-lightbox'];
+  const imageLightboxBody = `
+  <figure>
+  <amp-img id="img0"
+      srcset="
+          https://lh4.googleusercontent.com/-okOlNNHeoOc/VbYyrlFYFII/AAAAAAABYdA/La-3j3c-QQI/w2004-h1114-no/PANO_20150726_171347%257E2.jpg 2004w,
+          https://lh4.googleusercontent.com/-okOlNNHeoOc/VbYyrlFYFII/AAAAAAABYdA/La-3j3c-QQI/w1002-h557-no/PANO_20150726_171347%257E2.jpg 1002w"
+      width=527 height=293 layout="responsive"
+      on="tap:image-lightbox-1"
+      role="button" tabindex="0"></amp-img>
+  <figcaption>
+    This is a figcaption.
+  </figcaption>
+</figure>
+
+<amp-image-lightbox
+  id="image-lightbox-1"
+  layout="nodisplay"
+  data-close-button-aria-label="Close">
+</amp-image-lightbox>
+  `;
+  describes.integration('amp-image-lightbox opens', {
+    body: imageLightboxBody,
+    extensions,
+  }, env => {
+
+    let win;
+    beforeEach(() => {
+      win = env.win;
+    });
+
+    it('should activate on tap of source image', () => {
+      const lightbox = win.document.getElementById('image-lightbox-1');
+      expect(lightbox.style.display == 'none');
```

Comment:
looks like there's a travis error. It should be `expect(lightbox.style.display).to.equal('none')`